oka/chap2/knock12.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+
+
+def writing(col, elements, number):
+    tokens = [elements[i][number-1] for i in range(len(elements))]
+    with open(col, mode="w", encoding="utf-8") as f:
+        f.write("\n".join(tokens))
+
+
+path = "C:\\Users\\16t215\\Desktop\\hightemp.txt"
+col1 = "C:\\Users\\16t215\\Desktop\\col1.txt"
+col2 = "C:\\Users\\16t215\\Desktop\\col2.txt"
+with open(path, mode="r", encoding="utf-8") as f:
+    s = f.readlines()
+    elements = [s[i].split() for i in range(len(s))]
```

Comment:
`s`はリストだから，indexを取らなくてもいいんじゃないかな

```suggestion
    elements = [elem.split() for elem in s]
```